ports/glutin/window.rs

```diff
@@ -345,16 +345,16 @@ impl Window {
                     self.pending_key_event_char.set(Some(ch));
                 }
             }
-            Event::KeyboardInput(element_state, scan_code, Some(virtual_key_code)) => {
```

Comment:
Removing the match arm you linked should just be fine, why did you need to remove this `Some()`?